packages/asset-swapper/src/utils/market_operation_utils/path.ts

```diff
@@ -39,6 +39,7 @@ export class Path {
     public sourceFlags: number = 0;
     protected _size: PathSize = { input: ZERO_AMOUNT, output: ZERO_AMOUNT };
     protected _adjustedSize: PathSize = { input: ZERO_AMOUNT, output: ZERO_AMOUNT };
+    private _firstFill?: Fill;
```

Comment:
can we just use `this.fills[0]`?

Reply:
adjusted to use the BEST fill (we have some ETH2Dai tests which apparently the rate can get better with more fills somehow)